include/fmt/chrono.h

```diff
@@ -303,22 +309,17 @@ inline auto do_write(const std::tm& time, const std::locale& loc, char format,
 
 template <typename Char, typename OutputIt,
           FMT_ENABLE_IF(!std::is_same<Char, char>::value)>
-auto write(OutputIt out, const std::tm& time, const std::locale& loc,
-           char format, char modifier = 0) -> OutputIt {
-  auto str = do_write<Char>(time, loc, format, modifier);
+auto write_localized(OutputIt out, const std::tm& time, const std::locale& loc,
```

Comment:
Let's not rename this function because the "localized" part is clear from the signature (this function takes a locale).